naga/src/front/wgsl/parse/directive.rs

```diff
@@ -142,6 +126,7 @@ error: expected global declaration, but found a global directive
 
 ";
                 }
+                DirectiveKind::Unimplemented(kind) => match kind {},
```

Comment:
thought: I'm not sure how likely it is we'll have a new kind of directive to implement, so I wasn't sure if I should leave this in or not. I opted for not going to the effort, but I'd understand if it (a) seemed unlikely and (b) folks otherwise thought this was sorta gross. 😅

Reply:
Update: I have a commit now that removes this. Happy to add it as follow-up work, so it's easy to find and revert again. I could also be persuaded to add it here, but I think that that's less discoverable for reversion later.

Reply:
Yes I think we should remove this now that it's effectively dead code. If you want to do it in a different PR that's fine as well.

Reply:
Broken out to [`wgpu`#6463](https://github.com/gfx-rs/wgpu/pull/6463).